module/VuFind/src/VuFind/Db/Entity/AccessToken.php

```diff
@@ -38,54 +38,71 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="access_token")
+ * @ORM\Table(name="access_token",
+ * indexes={@ORM\Index(name="user_id", columns={"user_id"})}
+ * )
  * @ORM\Entity
  */
 class AccessToken implements EntityInterface
 {
     /**
+     * Unique ID.
+     *
      * @var string
      *
-     * @ORM\Column(name="id", type="string", length=255, nullable=false)
+     * @ORM\Column(type="string", length=255, nullable=false)
```

Comment:
Why was the name removed here? Is this just an optimization since the name will default to the property name if not specified, or is there another reason? (I don't think this is wrong, just curious about the particular reason since I'm also still learning Doctrine).